pkg/handler/dashboard/dashboard.go

```diff
@@ -72,12 +72,12 @@ func (h *handler) GetProjectSizes(c *gin.Context) {
 // @Tags Dashboard
 // @Accept json
 // @Produce json
-// @Param Authorization header string true "jwt token"
+// @Security BearerAuth
 // @Param projectID   query  string false  "Project ID"
 // @Success 200 {object} view.WorkSurveyResponse
-// @Failure 400 {object} view.ErrorResponse
-// @Failure 404 {object} view.ErrorResponse
-// @Failure 500 {object} view.ErrorResponse
+// @Failure 400 {object} ErrorResponse
+// @Failure 404 {object} ErrorResponse
```

Comment:
missing query params

Reply:
this PR just fix to the "client" so we should ignore d.., e..., f... folder